doc/cephadm/terminology.rst

```diff
@@ -0,0 +1,49 @@
+.. _cephadm-termnilogy:
+
+============
+Termninology
+============
+
+
+`Cephadm` introduces quite a few new terms that need clear demarcations. This article is supposed to go over them and
+explain what they are all about.
```

Comment:
Demarcations makes sense, but it's not great for such a global audience. Let's go for something like....